pkgs/development/tools/poetry2nix/poetry2nix/overrides/default.nix

```
@@ -2410,7 +2410,7 @@ lib.composeManyExtensions [
 
       # Stop infinite recursion by using bootstrapped pkg from nixpkgs
       bootstrapped-pip = super.bootstrapped-pip.override {
-        wheel = (pkgs.python3.pkgs.override {
+        wheel = (pkgs.python.pkgs.override {
           python = self.python;
         }).wheel;
```

`pkgs.python` is an alias for `python2` but some of poetry2nix's users want `python3`. I think you want
```suggestion
        wheel = self.python.pkgs.wheel;
```

---

Fixed.